docs/src/main/asciidoc/centralized-log-management.adoc

```diff
@@ -103,11 +102,12 @@ version: '3.2'
 
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:{es-version}
+    image: {elasticsearch-image}
```

Comment:
We need to include this attribute here:
https://github.com/quarkusio/quarkusio.github.io/blob/develop/_guides/attributes.adoc

Let's do it now so that we don't forget about it.

Reply:
Here you go: https://github.com/quarkusio/quarkusio.github.io/pull/1438

I don't really know what I'm doing though; I would have expected this file to be generated. Should I remember to keep it in sync every time we upgrade to a new version of Elasticsearch?

Reply:
Yeah unfortunately it's not generated. It wasn't when I took over the role and I never took the time to think about how to do this properly.